frontend/src/components/UserProfile/UserEmailList.tsx

```diff
@@ -178,6 +172,14 @@ const UserEmailList: React.FC<{
           onRemove={onRemove}
         />
       ))}
+      {(prevPage || nextPage) && (
```

Comment:
I wonder if it would be good to still display a total when there is just one page?